### ISSUE TITLE
German translation available

### ISSUE THREAD

**Issue by deus-e-x** [TheFantasticWarrior/chrome-extension-imagus#60]
Hi there TheFantasticWarrior,

I created a German translation for your Imagus mod plugin. Please let me know whether you would like to add it to the plugin and I'll post it here as an attachment or make it available to you by means you prefer.

Best regards
deus-ex

**Comment by TheFantasticWarrior** [comment]
Can you make the file like one of https://github.com/TheFantasticWarrior/chrome-extension-imagus/tree/new/src/_locales

I made some new changes like 
"SIV_BUTT_KEEP": {
        "message": "View the keep list when nothing is selected, or add/remove selected items."
    },
    "SIEVEKEEPADV_TIP": {
        "message": "ON: Compare new and old Sieve to decide what to keep, able to keep custom Sieve or other changes. OFF: Keep only on/off state, all other modifications will be deleted. Green means the sieve is in the list to keep. If you made changes and it's not green, you need to manually add it."
    },
    "SIV_KEEP": { "message": "Add to Keep list" },

**Comment by deus-e-x** [comment]
Yes, sure thing. I already had it available in this format, I used the English messages.json from the plugin distribution as a template.

I have just added your latest changes/additions, as you just hinted.  Find the German translation attached as a Zip-archive. Let me know if you need any changes/updates.

Cheers,
deus-ex

[deus-ex - Imagus mod v0.10.15+ German translation including latest updates.zip](https://github.com/TheFantasticWarrior/chrome-extension-imagus/files/14101011/deus-ex.-.Imagus.mod.v0.10.15%2B.German.translation.including.latest.updates.zip)

**Comment by TheFantasticWarrior** [comment]
some characters appear as error, can you make sure the encoding is UTF-8?

**Comment by deus-e-x** [comment]
Sure, here's the updated file all UTF-8 encoded.

[deus-ex - Imagus mod v0.10.15+ German by deus-ex_UTF-8 encoding.zip](https://github.com/TheFantasticWarrior/chrome-extension-imagus/files/14120475/deus-ex.-.Imagus.mod.v0.10.15%2B.German.by.deus-ex_UTF-8.encoding.zip)

**Comment by deus-e-x** [comment]
1. The German translation isn't listed on the info page.
2. There doesn't seem to be an option to set the language.

**Comment by TheFantasticWarrior** [comment]
1. sorry I forgot
2. it is in firefox language options

**Comment by deus-e-x** [comment]
RE 2: Could you add an option to override Firefox/Chrome language setting, allowing one to choose from the plugin's built-in translations?

For example, I have my system/browser language all set to English, so I'd prefer a convenient way to see my translation easily, without having to alter the system/browser settings.
